admin/scripts/sync_issues.py

```diff
@@ -0,0 +1,214 @@
+#!/bin/python
```

Comment:
It would be great to document the format of data which is expected by this script. Maybe just having some example data is enough.